ci/docker-compose-build.yaml

```diff
@@ -17,11 +17,20 @@ version: '3.5'
 services:
   tcf:
     image: tcf-ci-build:${ISOLATION_ID}
-
     build:
       context: ..
       dockerfile: docker/Dockerfile.tcf-dev
       args:
         - http_proxy
         - https_proxy
         - no_proxy
+
+  avalon-lmdb:
+    image: avalon-lmdb:${ISOLATION_ID}
+    build:
+      context: ..
+      dockerfile: examples/shared_kv_storage/Dockerfile
```

Comment:
My understanding is we agreed to move shared_hv_storage to top level common and move out of examples ?